src/System.CodeDom/ref/System.CodeDom.cs

```diff
@@ -58,7 +58,7 @@ public partial class CodeArrayCreateExpression : System.CodeDom.CodeExpression
         public CodeArrayCreateExpression(System.Type createType, int size) { }
         public System.CodeDom.CodeTypeReference CreateType { get { throw null; } set { } }
         public System.CodeDom.CodeExpressionCollection Initializers { get { throw null; } }
-        public int Size { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
+        public int Size { get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
```

Comment:
Do we really need the `CompilerGeneratedAttribute` attribute for each property getter/setter?

cc @@buyaa-n, @krwq

Reply:
@ericstj, thoughts on explicitly listing these attributes in the ref `[System.Runtime.CompilerServices.CompilerGeneratedAttribute]`? Are they required or are they an artifact of some previous tooling/sources?